### PR TITLE
CORTX-30785: Fixing the Alex bug

### PR DESCRIPTION
**Pull request by venkuppu-chn**
Signed-off-by: Venkatesh K <venkatesh.k@seagate.com>

## Description
Alex's Github Action is fails with an error "reviewdog: PullRequest needs 'git' command: failed to run 'git rev-parse --show-prefix'" 

## Type of change
Changed the permissions of the directory where the target repository is checkeout in the workflow of GitHub Actions.

- [x] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds new functionality)
- [ ] Breaking change (bug fix or new feature that breaks existing functionality)
- [ ] Third-party dependency update
- [ ] Documentation additions or improvements
- [ ] Code quality improvements to existing code or test additions/updates

## Applicable issues

- This change fixes an issue: #
- This change is related to an issue: #

## CORTX image version requirements

This change requires the following images:

- `cortx-all:<version>`
- `cortx-rgw:<version>`

## Checklist

- [ ] The change is tested and works locally.
- [ ] New or changed settings in the solution YAML are documented clearly in the README.md file.
- [ ] All commits are signed off and are in agreement with the [CORTX Community DCO and CLA policy](https://github.com/Seagate/cortx/blob/main/doc/dco_cla.md).

If this change addresses a CORTX Jira issue:

- [ ] The title of the PR starts with the issue ID (e.g. `CORTX-XXXXX:`)

### CODE REVIEW

File: .github/workflows/alex_reviewdog.yml [hunk 1]
```diff
@@ -25,7 +25,6 @@ on:
     types: [ opened, synchronize ]
     branches:
       - main
-      - integration
```
Comment (keithpine) [inline .github/workflows/alex_reviewdog.yml:28]: Why has this branch been removed? All of our PRs are based against `integration`.

Comment (mukul-seagate11) [reply .github/workflows/alex_reviewdog.yml:28]: @venkuppu-chn, please address the comment as raised

File: .github/workflows/alex_reviewdog.yml [hunk 2]
```diff
@@ -24,7 +24,6 @@ on:
   pull_request:
     types: [ opened, synchronize ]
     branches:
-      - main
       - integration
```
Comment (mukul-seagate11) [inline .github/workflows/alex_reviewdog.yml:27]: @venkuppu-chn, add both the branches i.e. main & integration in the workflow

**Review by mukul-seagate11 — CHANGES_REQUESTED** [review]

**Review by osowski — CHANGES_REQUESTED** [review]
Please use both `main` and `integration` branches in the trigger section.

**Comment by venkuppu-chn** [comment]
@osowski and @mukul-seagate11 We have to create a another PR against main branch if we required Alex workflow for main branch. Since, this PR will land on integration branch so there no point to add main branch in this PR.

File: .github/workflows/alex_reviewdog.yml (hunk 2 above)
Comment (mukul-seagate11) [reply .github/workflows/alex_reviewdog.yml:27]: another PR will be raised for Alex workflow with `main` branch

**Review by mukul-seagate11 — APPROVED** [review]
looks good as another PR will be raised for Alex workflow with main branch

**Comment by mukul-seagate11** [comment]
let me raise new PR with changes

**Pull request closed without merging**